util/find_missing_translations.ts

```diff
@@ -116,7 +116,7 @@ const parseJavascriptFile = (file: string, inputLocales: Lang[]) => {
 
         const openStr = openMatch[2];
 
-        if (!openStr)
+        if (openStr === undefined || openStr.length === 0)
```

Comment:
```suggestion
        if (openStr === undefined)
```

This match is always of at least length 1 because it includes the curly brace.